src/tools/illink/test/Mono.Linker.Tests.Cases/DataFlow/DynamicObjects.cs

```diff
@@ -134,7 +126,6 @@ class DynamicInRequiresUnreferencedCodeClass
 			[RequiresUnreferencedCode("message")]
 			class ClassWithRequires
 			{
-				// Analyzer hole: https://github.com/dotnet/runtime/issues/94057
 				[ExpectedWarning ("IL3050", ProducedBy = Tool.NativeAot)]
 				public static void MethodWithDynamicArg (dynamic arg)
```

Comment:
Should we have a new issue describing the need to fix this for RDC as well (eventually)?
or maybe I'm missing something - I would expect analyzer to behave the same as NativeAOT - or if it doesn't some comment or issue link describing the difference and "why".